IHP/Controller/Session.hs

```diff
@@ -53,6 +56,12 @@ getSessionUUID name = do
             Just (Just value) -> Just value
             _                 -> Nothing
 
+basicAuth :: (?requestContext::RequestContext) => Text -> Text -> Text -> IO ()
```

Comment:
Can we add some haddock documentaton `| -- ...` to this function? This is then visible when using e.g. haskell language server.

Reply:
Added now.

Reply:
..and that hunter2 reference still makes me chuckle :)

Reply:
finally someone got it haha :D